xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -241,6 +241,8 @@ std::string CGUIWindowMusicNav::GetQuickpathName(const std::string& strPath) con
     return "Singles";
   else if (path == "special://musicplaylists/")
     return "Playlists";
+  else if (path == "musicdb://discs/")
+    return "discs";
```

Comment:
CGUIWindowMusicNav::GetQuickpathName is deprecated function so can remove this change - see PR16626